client/src/views/TableDetail.vue

```diff
@@ -66,10 +66,16 @@
       </v-app-bar>
       <div class="wrapper">
         <v-data-table
+          fixed-header
+          height="calc(100vh - 123px)"
           class="table-details"
           :headers="dataTableHeaders"
           :items="dataTableRows"
-          :items-per-page="15"
+          :items-per-page="20"
+          :footer-props="{
+            itemsPerPageOptions: [5, 10, 15, 20, -1],
+            showFirstLastPage: true,
+          }"
```

Comment:
Can you revert this? I am taking care of this paging functionality in #247, which will likely conflict with these changes.

What does `showFirstLastPage` do?

Reply:
Just for completeness, I spoke to @jtomeck IRL and learned that `showFirstLastPage` simply includes the first and last page buttons in addition to previous and next in the paginator.